python/ola/ClientWrapperTest.py

```diff
@@ -173,6 +173,7 @@ def d():
     wrapper.AddEvent(datetime.timedelta(milliseconds=5), b)
     wrapper.AddEvent(10, c)
 
+    # Nothing is called yet
```

Comment:
Probably a little clearer:
```suggestion
    # Nothing has been called yet
```